content/2022/01/minFraud.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="251.20136" height="171.22353" viewBox="0 0 154.5 102.5">
```

Comment:
Did you get the image to work? I'm not seeing it show up in the staging environment or locally.

Reply:
i did, but i wonder if i didn't push the changes. i will double check.

Reply:
it works when i preview it on my machine, and there are no additional changes to commit.

Reply:
And you see it here? https://mm-blog-site-staging--pr-18-tq160xsp.web.app/